packages/svgo-jsx/svgo-jsx.js

```diff
@@ -96,7 +96,7 @@ const convertAttributes = (node, svgProps) => {
       props.set(name, JSON.stringify(value));
     }
   }
-  if ((node.name === "svg" || node.name === "Svg") && svgProps) {
+  if (parent && parent.type === "root" && svgProps) {
```

Comment:
It's never null. Not sure how to enforce it with types but there is no types here yet.